api/handlers/find_user.go

```diff
@@ -16,21 +15,28 @@ type findUser struct{
 	rt *ShivaniCustomServerExample1.Runtime
 }
 func (f *findUser) Handle(fup users.FindUsersParams) middleware.Responder{
-	// fup.Name
-	// v:=service.NewManager()
-	// v.ListUsers()
-	
-	fmt.Println("Application Name is",f.rt.GetApplicationName())
-	// n:="Shivani Sharma"
-	// us:= []*models.User{{Address:"ABC",ID:2,Name:&n}}
-	
-	us,_:= f.rt.GetManager().ListUsers(fup.Name,fup.Limit)
+
+	var limit int32
+	// var name string
+	var us []*domain.User
```

Comment:
not required. use initialization concept like 
us, err := <func call >

Reply:
Done ! Please check